frontend/src/components/Table/Table.tsx

```diff
@@ -143,31 +146,73 @@ export interface TableProps<T extends object = {}, TFilter extends object = {}>
   }) => React.ReactNode;
 }
 
-const IndeterminateCheckbox = React.forwardRef(({ indeterminate, ...rest }: any, ref) => {
-  const defaultRef = React.useRef();
-  const resolvedRef: any = ref || defaultRef;
-
-  React.useEffect(() => {
-    if (resolvedRef.current) {
-      resolvedRef.current.indeterminate = indeterminate;
+const IndeterminateCheckbox = React.forwardRef(
```

Comment:
Not sure what's going on with the latest changes.  Too advanced for me to read presently :(